apps/web/src/app/api/subscribe/route.ts

```diff
@@ -6,6 +6,13 @@ mailchimp.setConfig({
   server: process.env.MAILCHIMP_API_SERVER,
 });
 
+// TODO: This has got to be declared somewhere, right?
```

Comment:
[Question] I've never used the Mailchimp APIs or SDKs before, and couldn't find this in a quick scan of the `.d.ts`. Do they have a type for a caught error that includes the `.response`? If not, is there a place to feature request it?

Reply:
unsure(code is fairly old), tried to look at the docs and I dont even see this method anywhere anymore

https://mailchimp.com/developer/marketing/api/lists/

Reply:
Haha, fun. I'll just remove the TODO.